packages/drivers/odsp-driver/src/createFile/createFile.ts

```diff
@@ -193,10 +193,16 @@ export async function createNewEmptyFluidFile(
 			{ ...options, request: { url, method } },
 			"CreateNewFile",
 		);
+		const internalFarmType = checkForInternalFarmType(newFileInfo.siteUrl);
 
 		return PerformanceEvent.timedExecAsync(
 			logger,
-			{ eventName: "createNewEmptyFile" },
+			{
+				eventName: "createNewEmptyFile",
+				details: {
```

Comment:
Don't we stringify the details prop bag?

Reply:
Also, createFile has 2 functions. This function handles where we create the empty file and there is one where we create file with intial summary.

Reply:
> Don't we stringify the details prop bag?

It happens automatically now, no need to call stringify here.